enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -522,14 +522,31 @@ class SubsidyAccessPolicyCreditsAvailableResponseSerializer(SubsidyAccessPolicyR
         help_text='',
         source='subsidy_expiration_datetime',
     )
+    learner_content_assignments = serializers.SerializerMethodField('get_assignments_serializer')
+
+    def get_assignments_serializer(self, obj):
+        """
+        Return serialized assignments if the policy access method is of the 'assigned' type
+        """
+        if obj.access_method == AccessMethods.ASSIGNED:
+            assignments = obj.assignment_configuration.assignments.all().filter(
```

Comment:
Two minor things:
1. Nit: we don't need the `all()` before the `filter()` here :) 
2. Performance thing: the `LearnerContentAssignmentResponseSerializer` also serializes the actions related to each serialized assignment, so we could throw a `.prefetch('actions')` onto this queryset to help keep the number of queries for actions a constant (one).